CHANGELOG.md

```diff
@@ -33,6 +37,8 @@ Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and
 + Add - Drafts from a collection of precursor pipelines, including [DataJoint_Demo_DeepLabCut](https://github.com/MMathisLab/DataJoint_Demo_DeepLabCut) graciously provided by the Mathis Lab.
 + Add - Support for 2d single-animal models
 
+[0.3.0]: https://github.com/datajoint/element-deeplabcut/releases/tag/0.3.0
+[0.2.1]: https://github.com/datajoint/element-deeplabcut/releases/tag/0.2.1
```

Comment:
@CBroz1 Should we make a release of `0.2.1`?

Reply:
If we plan to merge/release here before the end of this week, I would just eliminate 0.2.1 from the changelog and move the mkdocs mention to under 0.3.0